src/MBrace.Streams/CloudStreams.fs

```diff
@@ -703,5 +703,54 @@ module CloudStream =
         sources
         |> ofCloudFiles CloudFileReader.ReadLines
         |> flatMap Stream.ofSeq
-        
+
+    /// <summary>
+    /// Constructs a CloudStream of lines from a path.
+    /// </summary>
+    /// <param name="path">The path to the text file.</param>
+    let ofTextFile (path : string) : CloudStream<string> =
```

Comment:
I suppose this should be ofTextFileByLines (or ofTextFileByLine and ofCloudFileByLine - both singular - I think singular reads better).